components/myComponent1/myComponent.js

```diff
@@ -0,0 +1,3 @@
+export default function Title() {
+  return <h1>Comestics Store</h1>;
```

Comment:
little typo in "Cosmetics"
```suggestion
  return <h1>Cosmetics Store</h1>;
```